elife/php-long-running-processes.sls

```diff
@@ -0,0 +1,47 @@
+# only supports 16.04!
+
+{% if pillar.elife.php.processes.enabled %}
+{% for name, configuration in pillar.elife.php.processes.configuration.items() %}
+{% set hyphenized = name | replace('_', '-') %}
+{% set service_name = salt['elife.project_name']() + '-' + hyphenized %}
+php-long-running-process-service-{{ hyphenized }}:
```

Comment:
That's quite a mouthful...

Reply:
`php-long-running-process` looks redundant?

Reply:
Want to absolutely avoid these states clashing with the ones of a project import this, any shorter unique prefix to use?

Reply:
Using https://github.com/elifesciences/builder-base-formula/pull/87/files#diff-ab2b69d66c8c8bcba9fd1952e9ab976aR177, `{{ hyphenized }}` is just `annotations` right? That doesn't look like the best name, should `annotations-queue-watch`? Prefixing that with `service` should be ok?

Reply:
Yeah, that should be `queue_watch` (the fact that pillar keys require `_` is the reason for the hyphenization)